frontend/src/main/java/starter/project/frontend/CarController.java

```diff
@@ -51,11 +50,7 @@ public Car getCar(@PathVariable("carId") int carId) {
     public Map<Integer, Car> batchUpdateCars(@RequestBody Map<Integer, Car> m) {
 
```

Comment:
I think, instead of looping through the map, you could just do:

`map.putAll(m);`

Reply:
thanks